dts/bindings/gpio/xlnx,xps-gpio-1.00.a.yaml

```diff
@@ -13,6 +13,9 @@ properties:
   reg:
     required: true
 
+  interrupts:
+    description: Interrupt for GPIO core, if enabled
+
```

Comment:
This is already described in `base.yaml`. If it is to be included here, the description needs to be much more detailed and specific to this IP core. Otherwise, please remove it.

Reply:
Removed